jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/Asn1EncodedDataRouter.java

```diff
@@ -75,6 +77,12 @@ public Asn1EncodedDataRouter(OdeProperties odeProperties) {
 
       this.asn1CommandManager = new Asn1CommandManager(odeProperties);
 
+      this.dataSigningEnabledRSU = System.getenv("DATA_SIGNING_ENABLED_RSU") != null && !System.getenv("DATA_SIGNING_ENABLED_RSU").isEmpty()
+            ? Boolean.parseBoolean(System.getenv("DATA_SIGNING_ENABLED_RSU"))
+            : false;
+      this.dataSigningEnabledSDW = System.getenv("DATA_SIGNING_ENABLED_SDW") != null && !System.getenv("DATA_SIGNING_ENABLED_SDW").isEmpty()
+            ? Boolean.parseBoolean(System.getenv("DATA_SIGNING_ENABLED_SDW"))
+            : false;
```

Comment:
Let's default the SDX signing to true, we want signed data flowing into that system. The RSU has the option of signing itself instead of here, but the SDX is just taking what data it's given.

Reply:
Updated to default to true